BeeSwiftToday/TodayTableViewCell.swift

```diff
@@ -73,12 +73,16 @@ class TodayTableViewCell: UITableViewCell {
             make.right.equalTo(-10)
         }
         
-        var addTitle = "Add"
-        // small screen hack
-        if (UIDevice.current.orientation == .landscapeRight || UIDevice.current.orientation == .landscapeLeft && UIScreen.main.bounds.height >= 375) ||
-           (UIScreen.main.bounds.width >= 375) {
-            addTitle = "Add data"
-        }
+        let addTitle: String = {
+            // small screen hack
+            if (UIDevice.current.orientation == .landscapeRight || UIDevice.current.orientation == .landscapeLeft && UIScreen.main.bounds.height >= 375) ||
+                (UIScreen.main.bounds.width >= 375) {
+                return "Add data"
+            } else {
+                return "Add"
+            }
+        }()
+        
```

Comment:
Were these changes supposed to be included in this PR?

Reply:
> Were these changes supposed to be included in this PR? 

Obviously do not address the 'username' instead of 'me' but probably a cleanup commit along the way, as part of the 

> Uncle Bob talks about the boy scout rule, which can be summarized as:
> **Leave the code cleaner than you found it.**
> Meaning that every time that we make a change in our code base, make an improvement in the code.


It is probably its own commit here, easy to revert / remove if it need be elsewhere.

Reply:
Commit 1a01dddcf6dfc7531319b0927d642fad7237bc55